modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-2419[OSSM-2419] Istiod pod starts but gets stuck in not ready state due to failed probes. Sometimes, when a mesh contains many member namespaces, the Istiod pod does not become ready due to a deadlock within Istiod. The deadlock has been resolved and the pod now starts as expected.
```

Comment:
The Jira issue above is private to Red Hat employees. If anyone else click on this link they will get an error.

Reply:
Marko and I thought it would be a good idea to create a public-facing version of this Jira entry to benefit customers who might hit the issue, so we created OSSM-3025 and removed 2419.

Done.